### PR TITLE
Update readme and dependencies

### PR DESCRIPTION
**Pull request by ehanson8**
### Helpful background context
Documentation updates to provide better description of the app and its context in acquisitions workflows

### Developer
- [x] All new ENV is documented in README (or there is none)
- [x] Stakeholder approval has been confirmed (or is not needed)

### Code Reviewer

- [ ] The commit message is clear and follows our guidelines (not just this pull request message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The documentation has been updated or is unnecessary
- [ ] The changes have been verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 A CLI application to generate and email credit card slips for Alma invoices via the Alma API.
 
+## Description
+Credit card slips are generated for items purchased with a credit card (e.g. from vendors like Amazon), which means we prepay rather than receive an invoice after shipment, which is our workflow with most vendors. The credit card slip is in lieu of a vendor-generated invoice, and is used for processing by Acquisition staff.
+
+ The application runs daily and retrieves PO lines with:
```
Comment (ghukill) [inline README.md:8]: Does "PO" stand for "Purchase Order"?  Not sure it needs spelling out, maybe obvious in context, but just curious.

**Review by ghukill — APPROVED** [review]
Not terribly familiar with the application, but given the scope of the PR around README only, looks good to me.

Left a comment, but not requiring changes.

File: README.md (hunk 1 above)
Comment (ehanson8) [reply README.md:8]: Correct! And yes, I think the context would be enough for future readers

Comment (jonavellecuerdo) [inline README.md:8]: Can you clarify where the 'raw' data is sourced from (i.e., where are the "PO lines" retrieved from?) 🤔

**Review by jonavellecuerdo — CHANGES_REQUESTED** [review]

**Comment by jonavellecuerdo** [comment]
I was also wondering if you think a document like the one we made for [Carbon](https://mitlibraries.atlassian.net/wiki/spaces/IN/pages/3455811597/Carbon+A+tool+for+loading+data+into+Symplectic+Elements) will eventually be created for this CLI app? 

For instance, with Carbon documentation--to prevent duplicating information--I decided to focus on making the README about the application (what it does, dev/testing, and deployment), while the Confluence doc goes into more detail on how the application is used (i.e., schedules). 

I don't think a document needs to be created right now, but I was curious to know what you thought of this approach!

**Comment by ehanson8** [comment]
I don't think that this app is nearly as complicated as `carbon`, which does need the more extensive documentation in my mind, but I will discuss with Christopher

**Review by jonavellecuerdo — APPROVED** [review]

**Pull request merged**